src/Resources/mappings/ShopUser.yaml

```diff
@@ -1,4 +1,35 @@
 Sylius\Component\Core\Model\ShopUser:
-    attributes:
+    properties:
         username:
-            faker: email
\ No newline at end of file
+            faker: email
+            unique: true
+        usernameCanonical:
+            faker: email
+            unique: true
+        salt:
+            faker: text
+            args: [60]
+        password:
+            faker: sha256
+        lastLogin:
+            faker: dateTime
+        emailVerificationToken:
+            faker: uuid
+        passwordResetToken:
```

Comment:
should be null.

Reply:
i can't put a null value for now, i will implement it in an other PR